6.9/alpine/Dockerfile

```diff
@@ -2,8 +2,10 @@ FROM alpine:3.4
 
 ENV NPM_CONFIG_LOGLEVEL info
 ENV NODE_VERSION 6.9.2
+ENV USER node
 
-RUN adduser -D -u 1000 node \
+RUN adduser -D -h /$USER -u 1000 -s /sbin/nologin -D $USER \
+    && apk update \
```

Comment:
You don't need to update the package index cache since we are using the `--no-cache` flag. The`--no-cache` flag tells apk to get the package index directly from the network, not the local cache.